app/client/components/FormRenderer.ts

```diff
@@ -860,6 +858,11 @@ class RefRenderer extends BaseFieldRenderer {
             },
             matchTriggerElemWidth: true,
           }),
+          css.resetSelectButton(
+            icon('CrossSmall'),
+            dom.hide((use) => !use(this.value)),
+            dom.on('click', (ev) => { this.value.set(''); ev.stopPropagation(); ev.preventDefault(); })
```

Comment:
One tiny improvement if you want, not that big of a deal though so as you wish: improving keyboard focus when we click the clear button.

Here, when we click the button, it gets hidden, so focus is kinda reset. We have to shift+tab to go back on the input. And assistive technologies may have trouble understanding correctly, since the thing is not there anymore.

One idea would be to focus back the select element when we click the clear button.